### PR TITLE
FCirc Multi 6 - Insignificant changes

### LINKED ISSUE #395: Update FCirc output format to accommodate data with multiple labels per tracer
### Feature Inspiration

Experiments that use tracers with multiple labels will have multiple Ra/Rd values per Animal, one for each labeled element. This requires an update to the output table, which right now only displays one set of values per serum sample.

### Feature Description

- Fcirc will need to incorporate labeled element into calculations, and display data from a single sample in a different row for each labeled element.
- Fcirc(intact) is split by element
- Fcirc(average) is split by element

Examples
* [google slides describing expected outputs](https://docs.google.com/presentation/d/1LjFfvoWVjKwN434eZFGzi2vHFO1P2VitOE8xNc92Cz4/edit?usp=sharing)
* [google sheet of example calculations](https://docs.google.com/spreadsheets/d/1OfwWCgYlLi3XH9QS62ekn-dlEyldwj7fV260Gp2yFjY/edit?usp=sharing)

### Comment

NOTE: This may be broken into multiple issues to track the changes required to the functions computing the values separately from the changes to the views.

-----

### Assumptions

- List of assumptions made WRT the code
- E.g. We will assume input is correct (explaining why there is no validation)

### Requirements

- List of conditions to be met for the feature
- E.g. Every column/row must display a value, i.e. cannot be empty

### Limitations

- A list of things this work will specifically not do
- E.g. This feature will only handle the most frequent use case X

### Affected/Changed Components

- Files
- Environment variables
- External executables
- Database tables
- Cron job settings
- Etc.

#### GUI Change description

Describe changes the user will see.

#### Code Change Description *(Pseudocode optional)*

Describe code changes planned for the feature.

#### Tests

A test should be planned for each requirement (above), where possible.

### LINKED ISSUE #398: Update FCirc output format to accommodate data with multiple tracer compounds per animal
### Feature Inspiration

Since Ra/Rd (FCirc) is dependent on the serum measurement of the tracer compound, experiments that use tracers with multiple labels will have multiple Ra/Rd values per serum sample, one for each tracer compound. This requires an update to the output table, which right now only displays one set of values per serum sample.

### Feature Description

- Fcirc reports values for each tracer compound as additional rows.

Recommendation from @mneinast, described in these [google drive slides](https://docs.google.com/presentation/d/1US08FxF7_hOIkqcKyRkYtLrZs1ZllsJ4MDsfdAd5Zck/edit?usp=sharing)

### Comment

-----

### Assumptions

- List of assumptions made WRT the code
- E.g. We will assume input is correct (explaining why there is no validation)

### Requirements

- List of conditions to be met for the feature
- E.g. Every column/row must display a value, i.e. cannot be empty

### Limitations

- A list of things this work will specifically not do
- E.g. This feature will only handle the most frequent use case X

### Affected/Changed Components

- Files
- Environment variables
- External executables
- Database tables
- Cron job settings
- Etc.

#### GUI Change description

Describe changes the user will see.

#### Code Change Description *(Pseudocode optional)*

Describe code changes planned for the feature.

#### Tests

A test should be planned for each requirement (above), where possible.

### PR DESCRIPTION
**Pull request by hepcat72**
## Summary Change Description

This is the junk drawer of changes in the FCirc multi effort.

## Affected Issue Numbers

- Resolves #395
- Resolves #398

## Code Review Notes

These changes don't need much scrutiny.

I turned off the CI tests for this PR.  These changes are just copies of files from the fcirc_multi branch dropped into a new branch off the `infusate_label` branch (the starting point of the master branch for this effort).  I am breaking that code review up into these smaller PRs that are separated by theme.  I am doing it by file simply because it's easier.  If any code here references something you cannot find, just consult to `fcirc_multi` branch/PR.

The PRs I will be submitting are:

- `fcirc_multi_models`
- `fcirc_multi_format`
- `fcirc_multi_validation`
- `fcirc_multi_pagination`
- `fcirc_multi_tests`
- `fcirc_multi_minor` (this PR)

## Checklist

These are the checks for the overall change set...

- [x] All issue requirements satisfied (or no linked issues)
- [x] [Linting passes](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#linting).
- [x] [Migrations created & committed *(or no model changes)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#migration-process)
- Tests
  - [ ] [Tests implemented *(or no code changes)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#test-implementation)
  - [x] [All *multi_working* tag tests pass locally](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#quality-control)
  - [x] All example load tests pass (remotely) *(or their failures predated and are unrelated to this branch)*
  - [x] `@tag("multi_working")` added to newly passing test functions/classes *(or no newly passing tests)*
  - [x] `@tag("multi_broken")`, `@tag("multi_unknown")`, or `@tag("multi_mixed")` removed from newly passing test functions/classes *(or no newly passing tests)*

### CODE REVIEW

File: DataRepo/formats/dataformat.py [hunk 1]
```diff
@@ -670,15 +670,22 @@ def getDistinctFields(self, order_by=None, assume_distinct=True, split_all=False
                             raise ae
 
             elif (
+                # TODO: The below or'ed conditions were already refactored once and it helps, but it should be
+                #       refactoeed again to simplify it more.  I think I added the second condition in order to re-use
+                #       code code for many-related I wanted to split that wasn't M:M.  That had caused a problem with
+                #       property fields, but I can't remember what.  But I also see that while I check for custom
+                #       distinct fields, I never use them.  That was probably some code I ended up refactoring, so that
+                #       can probably be removed.
```
Comment (lparsons) [inline DataRepo/formats/dataformat.py:678]: Move comment to issue.

File: DataRepo/models/tissue.py [hunk 2]
```diff
@@ -23,3 +23,6 @@ class Meta:
 
     def __str__(self):
         return str(self.name)
+
+    def is_serum(self):
+        return self.name.startswith(Tissue.SERUM_TISSUE_PREFIX)
```
Comment (lparsons) [inline DataRepo/models/tissue.py:28]: Nice.

**Review by lparsons — APPROVED** [review]
lgtm

File: DataRepo/formats/dataformat.py (hunk 1 above)
Comment (hepcat72) [reply DataRepo/formats/dataformat.py:678]: [I did awhile back](https://github.com/Princeton-LSI-ResearchComputing/tracebase/issues/484).

- [ ] Clean up the PR REVIEW notes and TODOs

**Review by fkang-pu — APPROVED** [review]

**Review by jcmatese — APPROVED** [review]

**Comment by hepcat72** [comment]
Will merge from #493

**Pull request closed without merging**